packages/web/components/Landing/Intro.tsx

```diff
@@ -1,90 +1,353 @@
-import { Button, Container, Heading, Stack, Text } from '@metafam/ds';
+import {
+  Box,
+  Button,
+  ChevronDownIcon,
+  Container,
+  Heading,
+  Image,
+  Stack,
+  Text,
+} from '@metafam/ds';
+// import { animated, config, useSpring } from '@react-spring/web';
+import BubbleLg from 'assets/landing/pretty/bubble-large.svg';
+import BubbleSm from 'assets/landing/pretty/bubble-small.svg';
 import BackgroundImage from 'assets/landing/sections/section-1.jpg';
 import { FullPageContainer } from 'components/Container';
-import { StartButton } from 'components/Landing/StartButton';
-import { useRouter } from 'next/router';
-import { useCallback, useEffect, useState } from 'react';
-import { BsArrowDown } from 'react-icons/bs';
+import { useEffect, useState } from 'react';
+
+import { upDownAnimation, upDownAnimationLong } from './animations';
 
 export const Intro: React.FC<{ currentSection: number }> = ({
   currentSection,
 }) => {
-  const router = useRouter();
+  // const router = useRouter();
   const [onScreen, setOnScreen] = useState(false);
+  const [showQuote, setShowQuote] = useState(false);
 
-  const handleSectionNav = useCallback(
-    (sectionId: string) => {
-      router.push(`#${sectionId}`);
-    },
-    [router],
-  );
+  // const slideStyles = useSpring({
+  //   config: config.stiff,
+  //   from: { transform: `translate3d(0, ${300}px, 0)` },
+  //   to: {
+  //     transform: `translate3d(0, ${showQuote ? -50 : 300}px, 0)`
+  //   }
+  // })
+
+  // const handleSectionNav = useCallback(
+  //   (sectionId: string) => {
+  //     router.push(`#${sectionId}`);
+  //   },
+  //   [router],
+  // );
```

Comment:
if these values are not needed anymore, why not delete them? could they become useful anytime in the future?

Reply:
That is the reason I leave things commented out as the commented code doesn't add to the build size but it *does* make the component messy and harder to read. Will remove. 🙏🏻